### DISCUSSION TITLE
Add Crypto Integration

### DISCUSSION THREAD

**Discussion by davve77** [Glench/ExtPay#115] [Ideas]
Crypto integration through [coinbase](https://www.coinbase.com/) or any other platform would be great

**Comment by jeffsdevinfo** [comment]
Yes also, for small fee transactions Stripe is expensive at 2.9% + $.30.  
This is literally $.33 on a $1 transaction fee.  
This is not including 5% on top of that for ExtPay.  
So $.38 for a $1 transaction.! Terrible.

**Reply by akump** [reply]
Yeah it adds up fast. You need to find a way for users to pay more upfront in order to avoid the fees. It's a shame really.
